includes/redis-connect-helper.php

```
@@ -32,159 +32,228 @@
+	$password    = $config['password'] ?? '';
+	$database    = isset( $config['database'] ) ? (int) $config['database'] : 0;
+	$timeout     = 0.5;
+	$errors      = array();
+
+	foreach ( $nodes as $node ) {
+		$parsed_node = wppo_parse_redis_node( $node );
+		$s_host      = $parsed_node['host'];
+		$s_port      = $parsed_node['port'];
+
+		try {
+			$sentinel = new \RedisSentinel(
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '128', column '20'). (undefined)

(MissingImport)